chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/dao/ChangeRequestStatusDAOImpl.java

```diff
@@ -34,9 +34,7 @@ public ChangeRequestStatus create(final ChangeRequest cr, final ChangeRequestSta
         String hql = "SELECT crStatus "
                 + "FROM ChangeRequestStatusEntity crStatus "
                 + "JOIN FETCH crStatus.changeRequestStatusType "
-                + "JOIN FETCH crStatus.certificationBody acb "
                 + "JOIN FETCH crStatus.userPermission "
-                + "JOIN FETCH acb.address "
```

Comment:
Without this address, when there's a change request that was updated by an ONC-ACB, and the job runs to generate a report of pending change requests, the populateDependentObjects method, on the "setStatuses" call, throws a "cannot create new session" error, because in the changeRequestStatusDAO, in the convert method, there's a conditional where if there is an ACB it tries to set the certification body value to be attached to that status. While this change might fix the "show history" part, it breaks the quartz report